pcsx2/gui/AppMain.cpp

```diff
@@ -546,19 +541,46 @@ void Pcsx2App::LogicalVsync()
 
 	FpsManager.DoFrame();
 	
-	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
-		if (EnableFMV == 1) {
-			Console.Warning("FMV on");
-			DoFmvSwitch();
-			EnableFMV = 0;
+	if (EmuConfig.Gamefixes.FMVinSoftwareHack || g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
```

Comment:
Please add back the DoFmvSwitch function. Add a on_off boolean parameter. This way code can factorized a little.